### PR TITLE
JSON Debug when adding media to the playlist

### PR DESCRIPTION
**Pull request by mudar**
Minor bug fix.
When adding multiple files to playlist, with non-exisiting ID for the last item, the result array had a trailing comma separator, which invalidated the JSON.
Now the API returns null if media file not found.

### CODE REVIEW

**Comment by rosenfeld** [comment]
Thanks, @mudar. @jwchile, would you mind providing @mudar commit-access to this repository? I don't have administrative rights over it so that I could do it myself...

Also, how could we update this plugin in the KDE official scripts page?

Thanks in advance

**Comment by mudar** [comment]
Thanks @rosenfeld for the merged push.

I'm not sure how to handle the next step, so I'll start by adding @jwchile 
I have done a big update, it's still in my fork, I didn't do a push (I guess I should and you could decline it!)

I have replaced iUI by jquery and jquery mobile, with an updated UI (Flat is in these days!). I have also renamed paths, added lyrics and covers, etc.. 

I'm happy with the result, still some minor JS improvements to do (not real bugs).

So basically I'm wondering if this should be merged or live sepearatly as a new project. Personnally, I prefer having it merged! But I'm refactorying/destroying other people's work, so it's not up to me... a matter of open-source etiquette ;-)

Thanks,
mudar
